examples/automotive/cheri/receive.cc

```diff
@@ -65,7 +65,7 @@ struct CarInfo
 // Driver structs/classes
 EthernetDevice      *ethernet;
 SonataLcd           *lcd;
-volatile SonataGPIO *gpio;
+volatile SonataGpioBoard *gpio;
```

Comment:
Nit: lines above are now mis-aligned

Reply:
I'm surprised the formatter let that through, I'll change this. I think it might be because checks are only running on `main` and not on this `staging` branch.

Reply:
Fixed after I ran `clang-format`, thanks for the spot!